.github/CODEOWNERS

```diff
@@ -28,6 +28,7 @@
 /convox/                                 @DataDog/agent-integrations
 /cortex/                                 @cortexapps/engineering support@getcortexapp.com
 /cyral/                                  @tyrannosaurus-becks product@cyral.com
+/embrace_mobile/                         @jpmunz @joage
```

Comment:
Can you also add an entry to the section below in this file?

Reply:
Yes!